obspy/taup/tests/test_tau.py

```diff
@@ -107,6 +107,28 @@ def test_p_iasp91_manual(self):
         self.assertAlmostEqual(p_arrival.purist_distance, 35.00, 2)
         self.assertEqual(p_arrival.purist_name, "P")
 
+    def test_p_iasp91_geo_manua(self):
```

Comment:
Some of these tests _ought_ to work without `geographiclib`. Specifically, we should get the same travel times, and the pierce points / paths should have the same distance-depth-time for the `_geo` and non geo variants of the methods. I think I can use the new `HAS_GEOGRAPHICLIB` module variable to do some sensible tests for this (i.e. test that the answers we can get in both cases do not depend on having `geographiclib` installed.

My fear at the moment is that obspy\geodetics is giving a different epicentral distance if `geographiclib` is or is not installed, and that is what is causing the tests to fail under AppVeyor... Or I've stuffed up the interface with  obspy\geodetics. In either case, sorting out the added tests should be instructive.